api/nakadi-event-bus-api.yaml

```diff
@@ -1647,6 +1695,18 @@ definitions:
         type: string
         description: |
           Offset of the event being pointed to.
+      shift:
```

Comment:
I mean my point is that shifting cursors is a very specific use-case and we actually don't want this to be widely used.